build/room_ensurer/room_ensurer.py

```diff
@@ -205,9 +210,42 @@ def _ensure_room_for_network(self, network: Networks, alias_fragment: str) -> No
                     for server_name, room_info in room_infos.items()
                 },
             )
+            return
+
+        # There are either mis
```

Comment:
I don't understand this comment.

Reply:
The comment is unnecessary. I removed it